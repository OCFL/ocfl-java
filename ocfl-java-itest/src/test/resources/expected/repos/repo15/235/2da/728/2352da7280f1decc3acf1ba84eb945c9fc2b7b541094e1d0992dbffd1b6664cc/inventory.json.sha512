40a7deef92d370a4d8cd797bc4d2d021be4aae4deeaaa272215a3a02e186ec4bb2de1e6250a2df613040bfcdb1e1e1b064b77190c1932bf3d8bca772c9cbdefa  inventory.json
