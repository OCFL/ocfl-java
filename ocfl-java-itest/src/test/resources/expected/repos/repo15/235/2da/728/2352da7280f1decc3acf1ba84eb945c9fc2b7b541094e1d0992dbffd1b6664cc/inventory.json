{
  "id" : "o1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v1",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "96a26e7629b55187f9ba3edc4acc940495d582093b8a88cb1f0303cf3399fe6b1f5283d76dfd561fc401a0cdf878c5aad9f2d6e7e2d9ceee678757bb5d95c39e" : [ "v1/content/file1" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "commit message",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "96a26e7629b55187f9ba3edc4acc940495d582093b8a88cb1f0303cf3399fe6b1f5283d76dfd561fc401a0cdf878c5aad9f2d6e7e2d9ceee678757bb5d95c39e" : [ "file1" ]
      }
    }
  }
}